src/main/kotlin/no/nav/syfo/oppfolgingsplanmottak/validation/FollowUpPlanValidator.kt

```diff
@@ -34,31 +45,64 @@ class FollowUpPlanValidator(
                 "employeeHasNotContributedToPlanDescription should not be set if employeeHasContributedToPlan = true"
             )
         }
-
-        val hasActiveSendtSykmelding =
-            sykmeldingService.hasActiveSentSykmelding(
-                employerOrgnr,
-                followUpPlanDTO.employeeIdentificationNumber,
-                isDev
-            )
-
-        validateEmployeeInformation(followUpPlanDTO.employeeIdentificationNumber, hasActiveSendtSykmelding)
     }
 
     private suspend fun validateEmployeeInformation(
-        employeeIdentificationNumber: String,
-        hasActiveSendtSykmelding: Boolean,
+        followUpPlanDTO: FollowUpPlanDTO,
+        employerOrgnr: String,
+        isDev: Boolean
     ) {
-        if (!employeeIdentificationNumber.matches(Regex("\\d{11}"))) {
+        if (!followUpPlanDTO.employeeIdentificationNumber.matches(Regex("\\d{11}"))) {
             throw FollowUpPlanDTOValidationException("Invalid employee identification number")
         }
 
+        if (isDev && followUpPlanDTO.employeeIdentificationNumber == "01898299631") {
+            return
+        }
+
+        val validOrgnumbers = validateArbeidsforhold(followUpPlanDTO, employerOrgnr)
+
+        validateSykmelding(followUpPlanDTO, validOrgnumbers)
+
+        if (pdlClient.getPersonInfo(followUpPlanDTO.employeeIdentificationNumber) == null) {
```

Comment:
Feilhåndterong i `PDLClienten` er kanskje designet ikke optimalt for denne løsningen. [Den ](https://github.com/navikt/lps-oppfolgingsplan-mottak/blob/14bc635a0b116b74f6a4babd7c50295333a73fd0/src/main/kotlin/no/nav/syfo/client/pdl/PdlClient.kt#L72-L102) returnerer `null` i alle tilfeller der henting ikke har skjedd og den burde kanskje kaste exception i tilfeller der der er feil ved henting vs `null` hvis personen ikke var funnet. 
 Ser nå med ferskt øye på [PDL-doken, og tror at kun `not_found`](https://pdl-docs.ansatt.nav.no/ekstern/index.html#_pdl_spesifikke_feil_fra_graphql) burde følge til ekte `EmployeeNotFoundException`.  Burde være annen feilmelding i andre tilfeller.

Reply:
Godt spottet. Det høres ut som vi bør endret det. Tenker du å ta det inn i denne PR-en, eller skal vi ta det i en egen?

Reply:
Vi har ingen sånne feil i prod siste måned, så best å gjøre det i egen PR. Og denne PRen er allerede stor. Jeg kan lage trello sak på dette. Så :shipit:  !